dsa/linear-search/linear_search.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import "fmt"
+
+func linearSearch(A []int, n int, x int) {
+	for i := 0; i < n; i++ {
+		if A[i] == x {
+			fmt.Println("Number found at position : ", i+1)
```

Comment:
If the array start from index 0, then it should return 4 if you are searching for 5 in [1,2,3,4,5,6,7,8,9,10]

Reply:
yeah! but for the user point of view 5 is in 5th place so making it as i+1

Reply:
Please return the index that is match the output from the issue

Reply:
done